CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "Chen Wang",
+    "authorEmail": "wang.chen@zoho.com",
+    "framework": "PyTorch",
+    "codeURL": "https://github.com/wang-chen/KervNets",
+    "model": "KervResNet34",
+    "hardware": "1 GPU (Nvidia GeForce GTX 1080 Ti)",
+    "costPerHour": "NA",
+    "timestamp": "2018-04-20",
+    "misc": {
+        "CUDA version": "8.0"
```

Comment:
What hyperparameters did you use? Please add the following information to `misc`:

- learning rate schedule
- batch size
- optimizer
- momentum
- weight decay

The [Google ResNet50 submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/master/ImageNet/train/google_resnet50_tpu_tensorflow18.json) is a good example for reference.